bitcoin-review-podcast/lightning-privacy-splice-panel.md

```diff
@@ -306,22 +318,22 @@ Let's put it this way on Lightning, right?
 I mean, if I am using my, you know, my Breeze or my Phoenix wallets, which is like, you know, just kind of like not fully custodial, but also not like your own node, It's kind of like in the middle there.
 And I'm paying for a beer at the Bitcoin park.
 Right.
-And they're using their whatever, like BTC pay server there.
+And they're using their whatever, like BTCPayserver there.
 You know, unless somebody is really, really trying to get me, the chances of anybody seeing that that's what happened are pretty much lower than if I was using on-chain transactions, right?
 
-Speaker 3: 00:16:21
+Vivek: 00:16:21
 
 I don't know if it's much lower, but I'd say it's lower.
 And there's definitely aggregators like Tony was saying, and that's always been the concern.
-I remember when Ellen Big showed up, everyone was like, oh, these guys are the feds or there's some other agency.
+I remember when Ellen Big showed up, everyone was like, oh!, these guys are the feds or there's some other agency.
 But you know, you just kinda, you're happy to get that inbound liquidity at the end of the day.
 I actually want to go higher level and just get our principles first, fundamentals established for the Lightning Network, the challenges for the privacy.
-I think I've heard T-Best and Tony explain it for those who haven't checked out their contributions, You should definitely check it out.
-T-Best has an amazing chapter in the Mastering Lightning book, which I recommend nonstop.
+I think I've heard Bastien Teinturier and Tony explain it for those who haven't checked out their contributions, You should definitely check it out.
+Bastien Teinturier has an amazing chapter in the Mastering Lightning book, which I recommend nonstop.
 And then Tony just sprung onto the scene, I think in 2021 with a massive privacy article describing issues with payment hashes, hash correlation, long paths.
-So from a high level approach, T-Bass, can you explain how, I guess, is it Bolt1 or something, the init messages with noisexk, basically how the sender is always unknown and the receiver used to be known and the challenges with that in regards to unannounced channels which were mistakenly called private channels back in the day.
+So from a high level approach, Bastien Teinturier, can you explain how, I guess, is it Bolt1 or something, the init messages with noisexk, basically how the sender is always unknown and the receiver used to be known and the challenges with that in regards to unannounced channels which were mistakenly called private channels back in the day.
```

Comment:
In this sentence, what is "noisexk"? 
In the next version of the Editor we will support "[unclear]" or "[inaudible]" tags that you will be able to add with one click when you can't understand a specific word. For now, you can add them manually, like
```suggestion
So from a high level approach, Bastien Teinturier, can you explain how, I guess, is it Bolt1 or something, the init messages with noisexk[unclear], basically how the sender is always unknown and the receiver used to be known and the challenges with that in regards to unannounced channels which were mistakenly called private channels back in the day.
```